ProductsWebApi/ProductsWebApi/Repository/ProductRepository.cs

```diff
@@ -8,30 +8,30 @@ namespace ProductsWebAPI.Repository
 {
     public class ProductRepository : IProductRepository
     {
-        private List<Product> _productItems;
+        private List<Products> _productItems;
```

Comment:
`Product` makes more sense. That is an entity.
`Products` feels like a list.
If they want they can have like
`List<Product> productList`.
You know what i mean?